build/eslint/xfail.list

```diff
@@ -178,10 +161,6 @@ apps/settings/js/panels/languages/languages.js # line 21, col 10, Error - Unsafe
 apps/settings/js/panels/simcard_manager/simcard_manager.js # line 296, col 6, Error - Unsafe assignment to innerHTML (no-unsafe-innerhtml/no-unsafe-innerhtml)
 apps/settings/js/panels/simpin/simpin.js # line 81, col 6, Error - Unsafe assignment to innerHTML (no-unsafe-innerhtml/no-unsafe-innerhtml)
 apps/sharedtest/test/unit/accessibility_helper_test.js # line 13, col 4, Error - Unsafe assignment to innerHTML (no-unsafe-innerhtml/no-unsafe-innerhtml)
-apps/sharedtest/test/unit/dialer/font_size_manager_test.js # line 106, col 10, Error - Unsafe assignment to innerHTML (no-unsafe-innerhtml/no-unsafe-innerhtml)
```

Comment:
Actually I forgot to fix it and since it was under the sharedtest directory I didn't notice in my local testing. The commit hook didn't complain either because the file was untouched so it didn't check it either. I'll fix it and update the PR.